src/test/java/net/sf/jabref/exporter/ExportFormatTest.java

```diff
@@ -23,75 +24,68 @@
 import net.sf.jabref.model.database.BibDatabase;
 import net.sf.jabref.model.entry.BibEntry;
 
+@RunWith(Parameterized.class)
 public class ExportFormatTest {
 
+    public IExportFormat exportFormat;
+    public String exportFormatName;
+    public static BibDatabase database;
+    public static Charset charset;
+    public static MetaData metaData;
+    public static List<BibEntry> entries;
+
+    public ExportFormatTest(IExportFormat format, String name) {
+        exportFormat = format;
+        exportFormatName = name;
+    }
+
+
     @Rule
     public TemporaryFolder testFolder = new TemporaryFolder();
 
 
-    @Before
-    public void setUp() {
-        Globals.prefs = JabRefPreferences.getInstance();
-        ExportFormats.initAllExports();
+    @BeforeClass
```

Comment:
Btw, is there a way to add a test which is not included in the coverage check? It would make sense to try to export a non-empty database for every export format just to make sure that no exceptions are thrown, but probably one wouldn't like to include these test in the coverage report...